### ISSUE TITLE
Saving support

### ISSUE THREAD

**Issue by bgrawi** [bgrawi/jsflap#2]
Add the ability to save to local storage or a regular file.

**Comment by sman591** [comment]
Or to a shareable URL, similar to schedule maker?

**Comment by bgrawi** [comment]
A shareable URL would require a server/backend, so that would have to be postponed until much later on. A local file/storage will be fine for now and I doubt anyone wants to actually share their automaton on Facebook or something :smile:

**Comment by sman591** [comment]
That's true. I'm thinking of the use case where I want to share what I have with a classmate; not necessarily everyone. But local file would work to start in order to avoid a backend
